### PR TITLE
Added EventManager.getCurrentGameNumber(); to return currentGameNumber

### PR DESCRIPTION
**Pull request by rstlnmage**
Added GameManager.matchProgressPointMultiplier(); to multiply base points according to Match progress. Modified awardPoints...(); to use multiplier

### CODE REVIEW

File: src/main/java/org/braekpo1nt/mctmanager/games/GameManager.java [hunk 1]
```diff
@@ -635,18 +635,44 @@ private void leavePlayersOnTeam(String teamName) {
...
+    /**
+     * Multiply base points according to match progress:
+     * Game 1 No multiplier
+     * Game 2 1.5 multiplier
+     * Game 3 1.5
+     * Game 4 2
+     * Game 5 2
+     * Game 6 2.5
+     * @param points Base points to be awarded
+     * @param points Rounded and cast to int
```
Comment (Braekpo1nt) [inline src/main/java/org/braekpo1nt/mctmanager/games/GameManager.java:648]: Redundant parameter comment

**Comment by Braekpo1nt** [comment]
Excellent work @rstlnmage ! This definitely gets the job done and it's a fine way to do it. However, I suggest a small change. Instead of putting the `matchProgressPointMultiplier()` method in `GameManager` and having `EventManager` expose its `currentGameNumber`, you should instead put `matchProgressPointMultiplier()` in the `EventManager`, and have the `GameManager` call `eventManager.matchProgressPointMultiplier(points)`. `EventManager.matchProgressPointMultiplier()` can then use the `currentGameNumber` directly. 

Further, instead of `matchProgressPointMultiplier()` taking in a `int points` value, doing the math, then returning the new value, I would prefer that it merely return the multiplier value. Then you would do the math in the `GameManager`, like so:

instead of...
```java
points = eventManager.matchProgressPointMultiplier(points);
addScore(points)
```

it would become...
```java
addScore(points * eventManager.matchProgressPointMultiplier());
```

The reason for this change is so that in the future, I can output the point multiplier factor to the users when they start the game:
```java
// somewhere in the EventManager when a new game is chosen:
messageAllParticipants("All points for this game are multiplied by " + matchProgressPointMultiplier());
```

**Comment by rstlnmage** [comment]
I can do this, however addScore() takes an int, so I would need to Math.round() it inline. Or, someday, we could change addScore everywhere to take floats, then change displaying the score as int.

**Comment by rstlnmage** [comment]
Moved to EventManager, returns current multiplier. Had to cast the math to (int) in addPoints() lines.

**Pull request merged**